onboard/nav/stateMachine.cpp

```diff
@@ -405,6 +493,25 @@ bool StateMachine::isWaypointReachable( double distance )
     return isLocationReachable( mPhoebe, mRoverConfig, distance, mRoverConfig["navThresholds"]["waypointDistance"].GetDouble());
 } // isWaypointReachable
 
+bool StateMachine::isRadioRepeaterDrop() const
+{
+    if ( mPhoebe->roverStatus().currentState() != NavState::RadioRepeaterTurn &&
+         mPhoebe->roverStatus().currentState() != NavState::RadioRepeaterDrive &&
+         mPhoebe->isDropRepeater() &&
+         mRepeaterDropComplete == false &&
+         mRadioRepeaterDrops == 0 )
+    {
+        Waypoint way = (mPhoebe->roverStatus().course().waypoints)[mCompletedWaypoints-1];
```

Comment:
I think you want to set `search` (and probably `gate`) to `false` here as well otherwise your logic will perform this if I'm following correctly.

Reply:
what do you mean by "set search and gate to false"? It can't go into either search state while in a Radio Repeater state.

Reply:
now I understand. Thank you!